.github/workflows/java-client-issue.yaml

```diff
@@ -0,0 +1,39 @@
+name: Slack notification for Java client issue closed
+on:
+  issues:
+    types:
+      - closed
+
+jobs:
+  slack-notify:
+    name: Send slack notification if java client issue has been closed
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'scope/clients-java' }}
```

Comment:
❓ I'm not 100% sure whether this works as I expect it to work. Is the label part of the event.

Reply:
could you verify it? Otherwise I'm also fine testing it once merged, always a bit tricky with testing the workflows manually before merge.

Reply:
No, unfortunately. My way would be to merge and fix any errors (#iterate).